guides/cookbook.md

```diff
@@ -533,6 +744,49 @@ console.log(String.fromCharCode.apply(null, writer.getBytes()));
 ```
 </div>
 
+<div class="tabpane Python" markdown="1">
+#### Using simpleion
+`dump/dumps` allow the user to provide the whitespace that will make up each
+indentation level via the `indent` parameter. Providing any amount of
+whitespace will cause text Ion to be pretty-printed.
+
+```python
+unformatted = u'{level1: {level2: {level3: "foo"}, x: 2}, y: [a,b,c]}'
+value = simpleion.loads(unformatted)
+pretty = simpleion.dumps(value, binary=False, indent=u'  ')
+print(pretty)
+```
+
+Result:
+```
+$ion_1_0
+{
+  level1: {
+    level2: {
+      level3: "foo"
+    },
+    x: 2
+  },
+  y: [
+    a,
+    b,
+    c
+  ]
+}
+```
```

Comment:
remove--this is redundant with the output shown just above the various pretty-printing recipes

Reply:
Done.